src/wp-includes/functions.php

```diff
@@ -6625,7 +6625,7 @@ function _wp_timezone_choice_usort_callback( $a, $b ) {
  *
  * @param string $selected_zone Selected timezone.
  * @param string $locale        Optional. Locale to load the timezones in. Default current site locale.
- * @return string
+ * @return string HTML select element of timezones.
```

Comment:
I prefer "for" instead of "of":
```suggestion
 * @return string HTML select element for timezones.
```